### PR TITLE
Test/profiletest

### PR DESCRIPTION
**Pull request by Jesuitman**
What I did:
Fixing the styling as well as making the profile view work

Did this break anything?

- [ ] No
- [x]  Yes

Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x]  Styling 
- [ ]  Bug fix (non-breaking change which fixes an issue)
- [ ]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ]  Super small fix (Corrected a typo, removed a comment, etc.)
- [ ]  Skip all the other stuff and briefly explain the fix.

Checklist:

- [ ]  If this code needs to be tested, all tests are passing
- [x]  The code runs locally
- [x]  There are comments where clarification/ organization is needed
- [ ]  The code is DRY. If it's not, I tried my best
- [x]  I reviewed my code before pushing

What's next?
error handling and testing

### CODE REVIEW

**Comment by MaryBruff** [comment]
Nice work! Glad we were able to get everything working and can move on to the last steps!

File: src/UserView/UserView.js [hunk 1]
```diff
@@ -1,59 +1,43 @@
...
-          )
-        )}
+    <div className="profile">
+      <div className="filter-buttons">
+        <button onClick={handleShowAll}>Show All</button>
+        <button onClick={handleShowFavorites}>Show Favorites</button>
+      </div>
+      <h2>Saved Controversies</h2>
+      <div>
+        {filteredControversies.map((controversy, index) => (
```
Comment (JoeyStewart) [inline src/UserView/UserView.js:28]: The solution to our struggles. Nice work

**Review by JoeyStewart — APPROVED** [review]

**Review by tenthwalker — APPROVED** [review]
Looks solid, I'll take a stab at adjusting the placement of login vs profile buttons after this merge

**Comment by MaryBruff** [comment]
Great work! Glad we were able to figure everything out!

**Pull request merged**